BurnOutSharp/Tools/Utilities.cs

```diff
@@ -253,6 +254,44 @@ private static FileVersionInfo GetFileVersionInfo(string file)
             }
         }
 
+        /// <summary>
+        /// Get the SHA1 hash of a file, if possible
+        /// </summary>
+        /// <param name="path">Path to the file to be hashed</param>
+        /// <returns>SHA1 hash as a string on success, null on error</returns>
+        public static string GetFileSHA1(string path)
+        {
+            if (path == null)
+                return null;
```

Comment:
`null` is not the only case where this would fail. Please use the following instead.

```suggestion
            if (string.IsNullOrWhiteSpace(path))
                return null;
```